src/main/java/com/loofah/graph/api/config/GraphAPIConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package com.loofah.graph.api.config;
+
+import com.loofah.graph.api.models.Skill;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class GraphAPIConfiguration {
+
+    @Bean
+    public List<Skill> setUpSkills() {
```

Comment:
I've added separate configs for setting up skills in the api and the tests - this was mainly me playing about with Spring - but may be useful when we come to set up our database.